hostinfo/hostinfo_linux.go

```diff
@@ -12,6 +12,7 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"regexp"
```

Comment:
I guess this is okay on Linux but we've tried to avoid this package because it brings in huge Unicode tables and makes binaries bigger.

But iOS now brings it in any for UPnP so maybe it's time to accept it.

Reply:
Replaced use of regex, we're looking for one fixed pattern.